### DISCUSSION TITLE
Add my first Account - I can't add my first account by Manual Entry

### DISCUSSION THREAD

**Discussion by namphamit95** [Authenticator-Extension/Authenticator#1266] [General]
Hi every one,

I add a new account using Manual Entry, although I have filled in the [Issuer] and [Secret] fields completely, nothing happens when I click the [OK] button. Could you please help me out? I'm using Google Chrome browser.

**Comment by Sneezry** [comment]
It's a known issue. We have a fix PR. See https://github.com/Authenticator-Extension/Authenticator/issues/1262#issuecomment-2301396138

**Reply by vizickles** [reply]
Hello Sneezry! Do you know approx. when this fix will be FR?
